### PR TITLE
add site anonystick

### PR DESCRIPTION
**Pull request by anonystick**

### CODE REVIEW

File: data.js [hunk 1]
```diff
@@ -3210,6 +3210,25 @@ module.exports = [
       "Linux",
     ],
   },
+  {
+    name: "Anonystick blog",
+    description: "A blog about backend technologies, tips and tricks technologies",
+    url: "https://anonystick.com",
+    twitter: "@TungNguyen07",
```
Comment (naresh-io) [inline data.js:3217]: It seems your twitter account is suspended, please add an active account.

File: data.js [hunk 2]
```diff
@@ -3210,6 +3210,25 @@ module.exports = [
...
+      "Redis",
+      "Elasticsearch",
+      "Mongodb",
+      "Web3",
+      "payments",
+      "Backend",
+      "Computer Science",
+      "Blockchain",
+      "Nodejs",
+    ]
```
Comment (naresh-io) [inline data.js:3230]: Please limit to maximum of 10 tags.

**Comment by AbbeyIT** [comment]
Hi @anonystick Any update on this pull request? Lmk if you need any help.

**Comment by markodenic** [comment]
Closing this due to inactivity. @anonystick, feel free to open a new PR.

**Pull request closed without merging**